modules/ROOT/partials/configuration/icon_list.adoc

```diff
@@ -58,7 +58,6 @@
 | `+embed-page+` | image:icons/embed-page.svg[embed-page.svg] | `+embed-page.svg+`
 | `+embed+` | image:icons/embed.svg[embed.svg] | `+embed.svg+`
 | `+emoji+` | image:icons/emoji.svg[emoji.svg] | `+emoji.svg+`
-| `+export+` | image:icons/export.svg[export.svg] | `+export.svg+`
```

Comment:
I am not 100% sure if the icon will be removed actually. It is still currently in the codebase

Reply:
Im not entirely sure, as this is the first time I have removed a plugin form the docs at this level.

Reply:
Removing it from here will only remove it from the icon_list in the documentation page for 7.0. "marking as resolved for now"